scheduler/src/cook/kubernetes/api.clj

```diff
@@ -133,7 +133,7 @@
           (handle-watch-updates all-pods-atom watch get-pod-namespaced-key
                                 callbacks)
           (catch Exception e
-            (log/error e "Error during pod watch for compute cluster" compute-cluster-name))
```

Comment:
Should we have a different clause for the expected exception and leave this as is?

Reply:
Good call, changed this to check the cause and only switch to `INFO` for `SocketTimeoutException`.